Airports/EGKK/SMR/Regions.txt

```diff
@@ -1269,6 +1330,84 @@ N051.08.58.195 W000.09.49.840
 N051.08.57.852 W000.09.49.958
 N051.08.57.565 W000.09.49.984
 N051.08.57.503 W000.09.49.645
+; - South_Terminal_Grass1
```

Comment:
I like these areas, should we add the long island between AN and Z and the circular island bordering Z, A, AN and M too?  I've been thinking about the green too -- they're still paved areas rather than grass so is it still appropriate? I've tried smrGrey which looked wrong because it was the same colour as the stands; what are your thoughts on smrBlue?

![image](https://user-images.githubusercontent.com/2829581/57585052-8e43a080-74da-11e9-98f9-b36d6b17da0f.png)